Mathlib/Data/ENNReal/Basic.lean

```diff
@@ -510,6 +510,16 @@ theorem toNNReal_nat (n : ℕ) : (n : ℝ≥0∞).toNNReal = n := by
 theorem toReal_nat (n : ℕ) : (n : ℝ≥0∞).toReal = n := by
   rw [← ENNReal.ofReal_natCast n, ENNReal.toReal_ofReal (Nat.cast_nonneg _)]
 
+lemma _root_.Nat.cast_toNNReal (N : ℕ) :
+    (Nat.cast N : ℝ≥0∞).toNNReal = (Nat.cast N : ℝ).toNNReal := by
+  rw [Real.toNNReal_of_nonneg (by positivity), ENNReal.toNNReal_nat]
+  congr
+
+lemma _root_.Nat.rpow_mul_toNNReal (n k : ℕ) (R : ℝ) :
```

Comment:
This seems very specific, maybe too much so.

Reply:
I'm happy to remove this. If another reviewer says do, I'll do it!

Reply:
Agreed, this (`Nat.rpow_mul_toNNReal`) is too specific and should be removed.

Reply:
Removed!